modules/govuk/manifests/apps/specialist_publisher_rebuild_standalone.pp

```diff
@@ -73,6 +87,10 @@
   $oauth_secret = undef,
   $publish_pre_production_finders = false,
   $publishing_api_bearer_token = undef,
+  $redis_host = undef,
+  $redis_port = '6379',
+  $redis_namespace = 'specialist-publisher-rebuild-standalone',
```

Comment:
If you use the govuk_sidekiq gem the namespace will be automatically configured to be the same as $GOVUK_APP_NAME